core/domain/rights_manager.py

```diff
@@ -140,6 +145,18 @@ def to_dict(self):
 
 
 def _get_activity_rights_from_model(activity_rights_model, activity_type):
+    """Constructs an ActivityRights object from the activity rights model.
+
+    Args:
+        activity_rights_model: ActivityRightsModel. Activity rights from the
+            datastore.
+        activity_type: The type of activity. Either one of these:
```

Comment:
Here and elsewhere: (i) add type information, (ii) consider rewording slightly:

    activity_type: str. The type of activity. Possible values:
        feconf.ACTIVITY_TYPE_EXPLORATION
        feconf.ACTIVITY_TYPE_COLLECTION